lib/alchemy/resource.rb

```diff
@@ -169,10 +169,22 @@ def attributes
           name: col.name,
           type: resource_column_type(col),
           relation: resource_relation(col.name),
-        }.delete_if { |_k, v| v.nil? }
+          enum: enum_values_collection_for_select(col.name),
+        }.delete_if { |_k, v| v.blank? }
       end.compact
     end
 
+    def enum_values_collection_for_select(column_name)
+      enum = model.defined_enums[column_name]
+      return if enum.blank?
+
+      enum.keys.map do |k|
+        [::I18n.translate!(k, scope: [:activerecord, :attributes, model.model_name.i18n_key, "#{column_name}_values"]), k]
```

Comment:
Can we use a `default` instead of raising?

Reply:
Maybe just `default: value.humanize`?

Reply:
yeah, that makes sense to me. I'll change that quickly.

Reply:
thanks